docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -519,19 +521,24 @@ PostgreSQL configuration is divided into two different subsections:
 
 These options are mutually exclusive, and can be turned on or off using the _enabled_ option. Exactly one of these two sections must have enabled = true, while the other must have enabled = false.
 
+Note that you can't change the `useExisting` setting after installation.
```

Comment:
it's postgres.useExisting.enabled and postgres.install.enabled. They can't change either of the settings mentioned above after initial install. Along with install.ldap_enabled.